packages/datadog-instrumentations/src/fs.js

```diff
@@ -0,0 +1,42 @@
+'use strict'
+
+const {
+  channel,
+  addHook
+} = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const fsChannel = channel('datadog:fs:access')
+const onePathMethods = ['access', 'appendFile', 'chmod', 'lchown', 'mkdir', 'mkdtemp', 'mkdtempSync', 'open',
+  'openSync', 'opendir', 'readdir', 'readFile', 'readlink', 'realpath', 'rm', 'rmdir', 'stat', 'truncate', 'unlink',
+  'utimes', 'writeFile', 'watch']
+const onePathMethodsSync = ['accessSync', 'appendFileSync', 'chmodSync', 'chownSync', 'lchownSync', 'mkdirSync',
+  'opendirSync', 'readdirSync', 'readFileSync', 'readlinkSync', 'realpathSync', 'rmSync', 'rmdirSync', 'statSync',
+  'truncateSync', 'unlinkSync', 'utimesSync', 'writeFileSync']
+const twoPathMethods = ['copyFile', 'link', 'rename', 'symlink']
+const twoPathMethodsSync = ['copyFileSync', 'linkSync', 'renameSync', 'symlinkSync']
+
+addHook({ name: 'fs' }, fs => {
+  shimmer.massWrap(fs, onePathMethods.concat(onePathMethodsSync), wrapFsMethod(fsChannel, 1))
+  shimmer.massWrap(fs, twoPathMethods.concat(twoPathMethodsSync), wrapFsMethod(fsChannel, 2))
+
+  shimmer.massWrap(fs.promises, onePathMethods, wrapFsMethod(fsChannel, 1))
+  shimmer.massWrap(fs.promises, twoPathMethods, wrapFsMethod(fsChannel, 2))
+  return fs
+})
+
+function wrapFsMethod (channel, numParams) {
+  function wrapMethod (fsMethod) {
+    return function () {
+      if (channel.hasSubscribers && arguments.length) {
+        const args = []
+        for (let i = 0; i < numParams; i++) {
+          args.push(arguments[i])
+        }
+        channel.publish(args)
```

Comment:
this closes the door to send more information in the future, but if we send an object, it will be able to increase in the future:
```suggestion
        channel.publish({ arguments: args })
```